gammacat/collection.py

```diff
@@ -81,16 +81,25 @@ def make_filename(self, meta, *, relative_to_index):
         else:
             path = self.out_path
 
-        path = path / 'data' / meta['reference_folder']
+        path = path / 'data' / meta['year']
 
-        tag = GammaCatStr.dataset_filename(meta)
+        tag = GammaCatStr.data_filename(meta)
 
         if meta['datatype'] == 'sed':
-            path = path / '{}_sed.ecsv'.format(tag)
+            if meta['file_id'] != 1:
```

Comment:
@pdeiml - Just FYI: there's a typo here, where you have "1" but it should be "-1", leading to wrong stuff:
```
	renamed:    output/data/2016ApJ...817L...7A/gammacat_2016ApJ...817L...7A_000014_lc.ecsv -> output/data/2016/2016ApJ...817L...7A/tev-000014-lc.ecsv
	renamed:    output/data/2016ApJ...817L...7A/gammacat_2016ApJ...817L...7A_000014_sed.ecsv -> output/data/2016/2016ApJ...817L...7A/tev-000014-sed--1.ecsv
	renamed:    output/data/2016ApJ...819..156B/gammacat_2016ApJ...819..156B_000049_lc.ecsv -> output/data/2016/2016ApJ...819..156B/tev-000049-lc-1.ecsv
	new file:   output/data/2016/2016ApJ...819..156B/tev-000049-sed-2.ecsv
	new file:   output/data/2016/2016ApJ...819..156B/tev-000049-sed-3.ecsv
	renamed:    output/data/2016ApJ...819..156B/gammacat_2016ApJ...819..156B_000049_sed.ecsv -> output/data/2016/2016ApJ...819..156B/tev-000049-sed-4.ecsv
	new file:   output/data/2016/2016ApJ...819..156B/tev-000049-sed.ecsv
	renamed:    output/data/2016ApJ...819..156B/gammacat_2016ApJ...819..156B_000049_ds.yaml -> output/data/2016/2016ApJ...819..156B/tev-000049.yaml
	renamed:    output/data/2016ApJ...821..129A/gammacat_2016ApJ...821..129A_000106_sed.ecsv -> output/data/2016/2016ApJ...821..129A/tev-000106-sed--1.ecsv
	renamed:    output/data/2016ApJ...821..129A/gammacat_2016ApJ...821..129A_000106_ds.yaml -> output/data/2016/2016ApJ...821..129A/tev-000106.yaml
	renamed:    output/data/2016ApJ...821..129A/gammacat_2016ApJ...821..129A_000110_sed.ecsv -> output/data/2016/2016ApJ...821..129A/tev-000110-sed--1.ecsv
	renamed:    output/data/2016ApJ...821..129A/gammacat_2016ApJ...821..129A_000110_ds.yaml -> output/data/2016/2016ApJ...821..129A/tev-000110.yaml
	renamed:    output/data/2016ApJ...821..129A/gammacat_2016ApJ...821..129A_000165_ds.yaml -> output/data/2016/2016ApJ...821..129A/tev-000165.yaml
	renamed:    output/data/2016MNRAS.459.2286A/gammacat_2016MNRAS.459.2286A_000043_ds.yaml -> output/data/2016/2016MNRAS.459.2286A/tev-000043.yaml
	renamed:    output/data/2016MNRAS.459.2550A/gammacat_2016MNRAS.459.2550A_000105_lc.ecsv -> output/data/2016/2016MNRAS.459.2550A/tev-000105-lc.ecsv
	renamed:    output/data/2016MNRAS.459.2550A/gammacat_2016MNRAS.459.2550A_000105_sed.ecsv -> output/data/2016/2016MNRAS.459.2550A/tev-000105-sed--1.ecsv
```